### ISSUE TITLE
Minecraft Prism launcher fails to launch the game with August 24's commits

### ISSUE THREAD

**Issue by ws909** [flathub/org.freedesktop.Platform.GL.nvidia#291]
Might be the same issue as https://github.com/flathub/org.freedesktop.Platform.GL.nvidia/issues/288, although I'm not sure if the issue is that the GPU is not found, or just an issue with GLX.

The fix to that issue does not resolve the problem I experienced, namely Minecraft launched from the Prism launcher failing with:
```
net.minecraft.class_1041$class_4716: GLFW error 65545: GLX: Failed to find a suitable GLXFBConfig
```
It was with the installation of that final commit that made me notice the error. I don't think I installed the two previous ones.

I downgraded to `b98fdca0728be9b1aa986b1569d4e15e79e26622fd3440e489feee9dda459afe` (same as in the original post in that other issue), and it works with that.

**Comment by guihkx** [comment]
Please share the output of the following commands to help us debug the problem:

```
cat /sys/module/nvidia/version
flatpak info "org.freedesktop.Platform.GL.nvidia-$(cat /sys/module/nvidia/version | tr '.' '-')"
flatpak info "org.freedesktop.Platform.GL32.nvidia-$(cat /sys/module/nvidia/version | tr '.' '-')"
```

**Comment by ws909** [comment]
```
555.58.02
```

```
          ID: org.freedesktop.Platform.GL.nvidia-555-58-02
         Ref: runtime/org.freedesktop.Platform.GL.nvidia-555-58-02/x86_64/1.4
        Arch: x86_64
      Branch: 1.4
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 2,1 MB

      Commit: b98fdca0728be9b1aa986b1569d4e15e79e26622fd3440e489feee9dda459afe
      Parent: 729273848144d076550474e6070e4e84e92fac582582df53e19b71439c776a70
     Subject: Add Vulkan 550.40.67 (30fddbb7)
        Date: 2024-08-07 22:34:21 +0000
```
         
```
          ID: org.freedesktop.Platform.GL32.nvidia-555-58-02
         Ref: runtime/org.freedesktop.Platform.GL32.nvidia-555-58-02/x86_64/1.4
        Arch: x86_64
      Branch: 1.4
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 1,8 MB

      Commit: 2e504bc006777ddb6117f2e63968e22349c54989c1ca3ba7d281ccae34ae3093
      Parent: e899b2cb2899d6b6cc20de30d68afb76123eb1d1818ea699709d15db12b99205
     Subject: Add Vulkan 550.40.67 (30fddbb7)
        Date: 2024-08-07 22:37:15 +0000
```

**Comment by guihkx** [comment]
It's really odd that the `b98fdca0728be9b1aa986b1569d4e15e79e26622fd3440e489feee9dda459afe` commit isn't broken for you, because it was broken for everyone else...

Can you please try doing this:

1. Close Minecraft/Prism
2. Run `flatpak --user update`
3. Recursively remove every `.ld.so` directory in `~/.var/app/<id>`:
  ```bash
  find ~/.var/app/ -maxdepth 2 -type d -name .ld.so -exec rm -Ir {} \;
  ```

And after that, try to launch Minecraft again.

**Comment by ws909** [comment]
Sure, but you'll have to wait a while. I'll report back later.

> It's really odd that the b98fdca0728be9b1aa986b1569d4e15e79e26622fd3440e489feee9dda459afe commit isn't broken for you, because it was broken for everyone else...

Seems that was the commit which worked for several others?

> Downgrading to b98fdca0728be9b1aa986b1569d4e15e79e26622fd3440e489feee9dda459afe fixes the issue.

**Comment by guihkx** [comment]
You're right, I misread the commit message. The commit you're on is "the good one".

Anyway, please try those steps when you can and report back.

**Comment by ws909** [comment]
Most peculiarly, `flatpak --user update` only listed GL32, not GL. Upon checking the GL version afterwards, it's now at the latest commit. Not sure what to get of that.

Anyway, followed step 3, then ran MC. It launched fine now. 

I did spend 3 hours fixing this yesterday (rather new to Linux, so I didn't even realize those were Flatpak updates at first (Pop!_Shop and Cosmic Store aren't particularly informative)), so I can ensure you it did not work then.

**Comment by guihkx** [comment]
> I'm not the only one that's had issues

Yeah there were indeed issues in the past few days with some of the latest driver updates, namely #283 and #288, which were then fixed by #285 and #289.

Now it should be normalized for everyone - assuming their Flatpak packages are up to date.

Anyway, since your issue has been solved, I'll close this one.
